src/_GuestBook/VIPGuest.java

```diff
@@ -0,0 +1,18 @@
+package _GuestBook;
+
+public class VIPGuest implements Guest{
+	VIPGuest(String name){
+		this.name = name;
+	}
+	String name;
+	
+	public String toString() {
+		return name;
+	}
+
+	@Override
+	public String getListing() {
+		// TODO Auto-generated method stub
```

Comment:
You'll have to return something here instead of `null` once you implement `buildNiceList()`!